docs/journey.js

```diff
@@ -143,51 +156,54 @@ function countryNameToCode(name) {
     
     $('#submitBtn').toggleClass("is-loading", true);
 
-    const country = Object.freeze(getCountryFromAutocomplete(autocomplete));
-    if (countryValid && country !== '') {
-      let queryResults = $('#entries');
-
-      journeyBoiye.add(
-        $('#activities').val().trim()
-      ).done(function(resp) {
-        min_rpi = resp.min_rpi;
-        max_rpi = resp.max_rpi;
-  
-        let code = countryNameToCode(country);
-        let iatas = resp.resultsArray.map(result => result.iata);
+    let msgs = runValidators(VALIDATORS);
+    if (msgs.length > 0) {
+      for (let msg of msgs) {
+        notify(msg, notifyLevel.DANGER);
+      }
 
-        flightPriceAvgs(code, iatas).then(function(avgsResp) {
-          let avgs = avgsResp.avgs;
+      $('#submitBtn').toggleClass("is-loading", false);
+      return;
+    }
 
-          let augmentedResults = resp.resultsArray.map(function(result, i) {
-            let avg = avgs[i];
-            result.fare = avg.avg;
-            result.fareValid = avg.success && avg.size > 0;
+    const country = Object.freeze(getCountryFromAutocomplete(autocomplete));
```

Comment:
Wasn't there an ECMA version issue with this? Should we have babel in this repo?

Reply:
I'm not sure what you are referring to, mate